### PR TITLE
Add additional references and descriptions to Test Configurations

### PR DESCRIPTION
**Pull request by titusfortner**
### Description
* Added links to the various Browser Specific Functionality
* Added descriptions and examples to the W3C options
* Added `username` and `accessKey` to list of valid capabilities
* Added section on Visual Testing with links to those documents

### Motivation and Context
Update the test configurations page with all valid capabilities and links to more information.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by titusfortner** [comment]
(if tests rely on secret values to run, then any PR from a fork will fail)

File: docs/dev/cli/test-configuration-options.md [hunk 1]
```diff
@@ -67,25 +82,90 @@ __Example__:
...
+the duration of the session. The default value is `false`.
+
+See the [WebDriver W3C Specification](https://w3c.github.io/webdriver/#dfn-insecure-tls-certificates) for more info.
+
+__Value Type__: boolean.
+
+__Example__:
+```java
+"acceptInsecureCerts": true
+```
```
Comment (kimsauce) [inline docs/dev/cli/test-configuration-options.md:99]: @titusfortner thanks for defining these!

**Review by kimsauce — APPROVED** [review]
Looks great to me -- i'll apply these changes. thank you so much! @titusfortner

**Comment by kimsauce** [comment]
Pushed these changes w/this PR https://github.com/saucelabs/sauce-docs/pull/409

**Pull request closed without merging**